src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -915,4 +915,42 @@ DocumentUpload findDocumentUploadInEvent(final ProcurementEvent event, final Str
             String.format(ERR_MSG_FMT_DOCUMENT_NOT_FOUND, documentId)));
   }
 
+  /**
+   * Export buyer attachments
+   * 
+   * @param procId
+   * @param eventId
+   * @param principal
+   * @return list of attachments
+   */
+  public List<DocumentAttachment> exportDocuments(final Integer procId, final String eventId,
+      final String principal) {
+    log.debug("Export all Documents from Event {}", eventId);
+    var event = validationService.validateProjectAndEventIds(procId, eventId);
+    var exportRfxResponse = jaggaerService.getRfx(event.getExternalEventId());
+    var status = jaggaerAPIConfig.getRfxStatusToTenderStatus()
+        .get(exportRfxResponse.getRfxSetting().getStatusCode());
+    var attachments = new ArrayList<DocumentAttachment>();
+
+    if (TenderStatus.ACTIVE != status) {
```

Comment:
Does `ACTIVE` map directly to `published` (just checking - I can't recall..) 👍

Reply:
Yes ACTIVE is mapped to published

https://standard.open-contracting.org/1.1/en/schema/codelists/#tender-status